ln-visualizer/exports.sh

```diff
@@ -0,0 +1,4 @@
+export APP_LNVISUALIZER_WEB_IP="10.21.21.98"
```

Comment:
As your app id is `ln-visualizer`, can you use this prefix for your env. vars. `APP_LN_VISUALIZER`.

Reply:
I agree that makes more sense. I've been pretty inconsistent with the naming so far, so thanks for calling this out.
I changed it to APP_LN_VISUALIZER